python/py-more-itertools/Portfile

```diff
@@ -37,10 +37,23 @@ if {${name} ne ${subport}} {
                             sha256  38a936c0a6d98a38bcc2d03fdaaedaba9f412879461dd2ceff8d37564d6522e4 \
                             size    67359
         depends_lib-append  port:py${python.version}-six
-    }
 
-    if {${python.version} < 37} {
         python.pep517       no
+    } elseif {${python.version} < 37} {
+        version             8.12.0
+        revision            0
+        epoch               1
+        distname            ${python.rootname}-${version}
+        checksums           rmd160  92e25b8097001dbf10dffced771e4c74a3d8923d \
+                            sha256  7dc6ad46f05f545f900dd59e8dfb4e84a4827b97b3cfecb175ea0c7d247f6064 \
+                            size    108698
+
+        python.pep517       no
```

Comment:
why do you restrict this to version 8.12.0 - according to [upstream](https://github.com/more-itertools/more-itertools/blob/e2f2d06366a33666c5e97194ad4d5d43e11b8dd9/pyproject.toml#L9) version 8.14.0 works for Python 3.5+, so I don't understand the need for this.

Reply:
@reneeotten because this is the last version which can be installed without PEP517. `8.14.0` for 3.5 installs nothing. You may check it here: https://ports.macports.org/port/py35-more-itertools/details/ ; the same had happened for 3.6: https://ports.macports.org/port/py36-more-itertools/details/

Reply:
the it should just be installed without PEP517 as has been the default for years; there shouldn't be a reason for downgrading the version

Reply:
@reneeotten I've tried to install `8.14.0` and `8.13.0` without PEP512 and it installed nothing. `8.12.0` the last release which can be installed without PEP512.

Reply:
@reneeotten it doesn't install anything because `setup.py` is empty at [8.14.0](https://github.com/more-itertools/more-itertools/blob/v8.14.0/setup.py) and [8.13.0](https://github.com/more-itertools/more-itertools/blob/v8.13.0/setup.py); anyway [8.12.0](https://github.com/more-itertools/more-itertools/blob/v8.12.0/setup.py) is different things.

It was removed as part of [switching to flit](https://github.com/more-itertools/more-itertools/commit/f4044df2e653662df4fbfd6c91a87e29922a42c2) and [re-created](https://github.com/more-itertools/more-itertools/commit/81df802cd7cba7f416210ab0e1bf17acd61d7d2c) after that.

So, technically speaking `8.12.0` is the last `py-more-itertools` without PEP517.

Reply:
okay, I am taking a look at this now. Python 3.6 builds fine with `8.14.0` while using `python.pep517` so there is no reason to restrict that one. I guess the remaining subport is than 3.5 - I'll take a look at that now.

Reply:
okay, for Python 3.5 they indeed messed this up. You can restrict it to an earlier version or probably patch back a working `setup.py` - I don't really care. We should be trying to remove these obsolete subports instead of putting too much work in to keep them working...

Reply:
My bad. 9.0.0 doesn't work on 3.6, but 8.14.0 works.